acid/features/history/model.py

```diff
@@ -59,34 +59,39 @@ def build_number(self):
         return None
 
     @classmethod
-    def get_branches(cls):
-        branches = select(b.ref for b in cls)
+    def get_all_branches(cls, pipeline):
```

Comment:
Did anyone use this method? And, maybe better name is like `get_branches_in_pipeline`?

Reply:
Update: I found! :smile: